app/Http/Controllers/DeleteAirlineController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Domain\Airline\Actions\DeleteAirlineAction;
+
+class DeleteAirlineController extends Controller
+{
+    public function __invoke(DeleteAirlineAction $deleteAirlineAction, $airlineId)
+    {
+        $deleteAirlineAction->execute($airlineId);
+    }
```

Comment:
```suggestion
    public function __invoke(Airline $airline): JsonResponse
    {
        $airline->delete();
        
        return response()->noContent();
    }
```
Remember to leverage Laravel's features whenever possible. In this case, [Route Model Binding](https://laravel.com/docs/10.x/routing). As for the action, same thing as in the `CreateAirlineController`. 
What the `response()->noContent()` does is generate a response with no content and the 204 HTTP status code. I personally do this for delete APIs as there's not much to be said after deleting something. The response may vary depending on the project and such.